src/database/migrations/0001_create_canonical_fields.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE canonical_fields (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  label VARCHAR NOT NULL,
```

Comment:
Thanks for humoring me on `varchar` : )

Reply:
Ha not at all!  Thank you for pointing out the benefits of avoiding TEXT